Sources/Rendering/Core/ScalarBarActor/test/testScalarBarNoExtras.js

```diff
@@ -0,0 +1,70 @@
+import test from 'tape-catch';
+import testUtils from 'vtk.js/Sources/Testing/testUtils';
+
+import 'vtk.js/Sources/Rendering/Misc/RenderingAPIs';
+import vtkRenderer from 'vtk.js/Sources/Rendering/Core/Renderer';
+import vtkRenderWindow from 'vtk.js/Sources/Rendering/Core/RenderWindow';
+
+import vtkScalarBarActor from 'vtk.js/Sources/Rendering/Core/ScalarBarActor';
+import vtkColorTransferFunction from 'vtk.js/Sources/Rendering/Core/ColorTransferFunction';
+
+import baselineNoExtras from './testScalarBarNoExtras.png';
```

Comment:
without extra, the scalarbar does not go as high as the case with extras.

Reply:
Is that an issue ? It was already the case before so I'm guessing it's intended